versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/power-automate.md

```diff
@@ -0,0 +1,246 @@
+
+#### Output
+
+You can use an output mapping to map the response:
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables. It comes with a pre-filled value of `= {statusCode: response.body.statuscode}`.
+
+Response example:
+
+```
+= {
+    "@odata.context": "https://[Organization URI]/api/data/v9.2/$metadata#flowsessions(statuscode,statecode,startedon,completedon)/$entity",
+    "@odata.etag": "W1276122",
+    "statuscode": 4,
+    "statecode": 0,
+    "startedon": "2022-06-16T12:54:40Z",
+    "completedon": "2022-06-16T12:57:46Z",
+}
+```
+
+Meaning of the different status codes based on the [official documentation](https://learn.microsoft.com/en-us/power-apps/developer/data-platform/reference/entities/flowsession#statuscode-choicesoptions):
+
+| Status | Statuscode   |
+| ------ | ------------ |
+| 0      | NotSpecified |
+| 1      | Paused       |
+| 2      | Running      |
+| 3      | Waiting      |
+| 4      | Succeeded    |
+| 5      | Skipped      |
+| 6      | Suspended    |
+| 7      | Cancelled    |
+| 8      | Failed       |
+| 9      | Faulted      |
+| 10     | TimedOut     |
+| 11     | Aborted      |
+| 12     | Ignored      |
+| 13     | Deleted      |
+| 14     | Terminated   |
+
+### Get flow outputs
+
+The operation **Get flow outputs** returns the output of the triggered Power Automate flow.
+
+#### Configuration
+
+For this section, you must fill out **Organization URI**. This is your dynamics 365 URL. [Visit the docs](https://learn.microsoft.com/en-us/power-apps/developer/data-platform/discovery-service) or [learn about environments](https://learn.microsoft.com/en-us/power-platform/admin/environments-overview#environment-details) to see how to find it.
+
+#### Input
+
+For this section, you must fill out **Flow session ID**. This is the unique identifier for entity instances. You can find this in the response of the 'Trigger a flow run' method.
+
+#### Output
+
+You can use an output mapping to map the response:
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables. It comes with a pre-filled value of `= {flowOutputs: response.body}`.
+
+Response example:
+
+```
+= {
+    "Output1": "My output value"
+}
+```
+
+## Appendix
+
+### Using Power Automate Connector best practice
+
+#### Using Webhook Connector as callback endpoint
+
+Oftentimes, it is desired to continue the process after a Power Automate flow run finished. You can use the **Callback URL** field if you select the **Trigger a flow run** to specify a URL which will be called after the flow is finished.
+
+If you wish to continue the Camunda process or start a new one, the [Webhook Connector](./http-webhook.md) is a good next step. For this, take the following steps:
```

Comment:
The build is currently failing because the `http-webhook` page was removed from 8.0 in #1789. This line, and line 219 below, should be adjusted to account for that.

Reply:
I replaced them with an info block that informs about webhook support in the newer version.